src/java.base/share/classes/jdk/internal/access/JavaLangReflectAccess.java

```diff
@@ -102,4 +102,7 @@ public void setConstructorAccessor(Constructor<?> c,
     /** Returns a new instance created by the given constructor with access check */
     public <T> T newInstance(Constructor<T> ctor, Object[] args, Class<?> caller)
         throws IllegalAccessException, InstantiationException, InvocationTargetException;
+
+    public Object invokeDefault(Object proxy, Method method, Object[] args, Class<?> caller)
+        throws Throwable;
```

Comment:
Minor nit: add a comment to the method so that it's consistent with the other JLRA methods.

Reply:
What about this:

```
    /** Invokes the given default method if the method's declaring interface is
     *  accessible to the given caller.  Otherwise, IllegalAccessException will
     *  be thrown.  If the caller is null, no access check is performed.
     */
    public Object invokeDefault(Object proxy, Method method, Object[] args, Class<?> caller)
        throws Throwable;
```

Reply:
> What about this:

Looks okay, main thing is to have it be consistent with the existing methods.